.pre-commit-config.yaml

```diff
@@ -65,6 +65,8 @@ repos:
       - id: mypy
         files: src
         args: []
```

Comment:
Perhaps it would be useful to run with the lowest / highest supported Python version (see e.g. https://github.com/scikit-hep/cabinetry/blob/aee8689cbdb56a0cf147bd38f4feb6f70a2345f5/.pre-commit-config.yaml#L8-L18).

Reply:
Added in pre-commit config